frontend/ui/network/manager.lua

```diff
@@ -119,6 +119,20 @@ function NetworkMgr:getProxyMenuTable()
     }
 end
 
+function NetworkMgr:getRestoreMenuTable()
+    return {
+        text = _("Auto restore wifi after wake-up"),
```

Comment:
Note that although I personally prefer the "wifi" spelling (because it's just a regular word now), we've standardized on "Wi-Fi". So perhaps something more like "Automatically restore Wi-Fi connection after resume"

Anyway, I don't know if this is really needed. My bad, I probably should've clarified but I thought the xkcd link made my point for me. :)

Of course on e.g. Android or your laptop, wifi turns itself back on automatically. So the KOReader way may not be intuitive. It's just that I usually don't use wifi, and when I do I like the fact that it automatically turns off when pressing stand-by. But I'm not saying it necessarily has to be that way.